### DISCUSSION TITLE
Feature Request: Ability to customize request options before it was sent to the server

### DISCUSSION THREAD

**Discussion by OzanKurt** [livewire/livewire#6783] [3. Feature ideas]
https://github.com/livewire/livewire/blob/7fbe81ce113a17bd92687174cd79a4791bd40764/js/request.js#L40-L71

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by OzanKurt** [comment]
Thanks!
